scripts/mbedtls_dev/ecp.py

```diff
@@ -682,3 +682,93 @@ def result(self) -> List[str]:
     @property
     def is_valid(self) -> bool:
         return True
+
+
+class EcpP448Raw(bignum_common.ModOperationCommon,
+                 EcpTarget):
+    """Test cases for ECP P448 fast reduction."""
+    symbol = "-"
+    test_function = "ecp_mod_p448"
+    test_name = "ecp_mod_p448"
+    input_style = "fixed"
+    arity = 1
+    dependencies = ["MBEDTLS_ECP_DP_CURVE448_ENABLED"]
+
+    moduli = [("fffffffffffffffffffffffffffffffffffffffffffffffffffffffe"
+               "ffffffffffffffffffffffffffffffffffffffffffffffffffffffff")] # type: List[str]
+
+    input_values = [
+        "0", "1",
+
+        # Modulus - 1
+        ("fffffffffffffffffffffffffffffffffffffffffffffffffffffffe"
+         "fffffffffffffffffffffffffffffffffffffffffffffffffffffffe"),
+
+        # Modulus + 1
+        ("ffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+         "00000000000000000000000000000000000000000000000000000000"),
+
+        # 2^448 - 1
+        ("ffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+         "ffffffffffffffffffffffffffffffffffffffffffffffffffffffff"),
+
+        # Maximum canonical P448 multiplication result
+        ("fffffffffffffffffffffffffffffffffffffffffffffffffffffffd"
+         "ffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+         "00000000000000000000000000000000000000000000000000000002"
+         "00000000000000000000000000000000000000000000000000000001"),
```

Comment:
The maximum canonical multiplication result is  $(P - 1)^2$  (where in this case $P = 2^{448} - 2^{224} - 1$)

```suggestion
        # Maximum canonical P448 multiplication result
        ("fffffffffffffffffffffffffffffffffffffffffffffffffffffffd"
         "fffffffffffffffffffffffffffffffffffffffffffffffffffffffd"
         "00000000000000000000000000000000000000000000000000000004"
         "00000000000000000000000000000000000000000000000000000004"),
```

Reply:
Sorry, that was indeed $(P )^2$ not $(P - 1)^2$